manager-rs/dauth-service/src/rpc/clients/backup_network.rs

```diff
@@ -33,7 +127,55 @@ pub async fn get_auth_vector(
     user_id: &str,
     address: &str,
 ) -> Result<AuthVectorRes, DauthError> {
-    todo!()
+    let response = context
```

Comment:
It would probably good to have some off-by default debug-level tracing/logging annotations into and out of these top-level handler functions, just so we can see the call flow in action in debug deployment!